tools/e2etesting/TestEventProcessor/TestEventProcessor.Businesslogic/TelemetryValidator.cs

```diff
@@ -40,6 +40,7 @@ public class TelemetryValidator : ITelemetryValidator
         private ValueChangeCounterPerNodeId _valueChangeCounterPerNodeId;
         private MissingValueChangesChecker _missingValueChangesChecker;
         private IncrementalIntValueChecker _incrementalIntValueChecker;
+        private SequenceNumberChecker _incrementalSequenceChecker;
```

Comment:
just an observation that is not strictly related to this pr. 
In this file there is a mix of coding styles regarding curly brackets. Sometimes the bracket is on the new line:
`public async Task<StartResult> StartAsync(ValidatorConfiguration configuration)`
`{`
sometimes is on the same line:
`if (_cancellationTokenSource != null) {`

I would like to know what do you think. Should we have to standardize the style when we made some changes to a file?

Reply:
I agree, we should be consistent with the coding guideline.